src/Symfony/Component/HttpKernel/Kernel.php

```diff
@@ -572,6 +572,9 @@ protected function getKernelParameters()
         }
 
         return array(
+            /**
+             * @deprecated since Symfony 3.3
```

Comment:
version looks wrong (otherwise, it would already be gone in 4.0).

And for that one, being able to warn when using the parameter looks useful.

Reply:
version is the right one, it was deprecated in 3.3, but we did not add the deprecation notice and didn't find time/energy/way to make it before 4.0. I can change to 4.2, no big deal.

Reply:
Moved to 4.2.